### DISCUSSION TITLE
Cuộc hội thoại đầu tiên :)

### DISCUSSION THREAD

**Discussion by nvatuan** [nvatuan/PBL6#2] [General]
Xin chào mọi người, mình là Phúc Trần Xuân, cùng với bạn Nguyễn Đăng Thông, là Team 9 - Fix som bugs. Hôm nay, chúng mình xin trình bày về mô hình scrum

**Comment by thongnd0200** [comment]
MK đã qua đây dạo chơi

**Comment by xuanphuc97** [comment]
Oke bro

**Comment by xuanphuc97** [comment]
Xin chào các bạn tôi đã làm xong mấy cái demo.

**Comment by nvatuan** [comment]
Xin chào tát cả các bạn

**Comment by thongnd0200** [comment]
Xin chào cô và các bạn
